config.json

```diff
@@ -12,6 +12,20 @@
   "version": 3,
   "online_editor": null,
   "test_pattern": "_test\\.dart$",
+  "files": {
+    "solution": [
+      "lib/%{snake_slug}.dart"
```

Comment:
https://github.com/exercism/docs/pull/58#issuecomment-780789579 mad me concerned, what happens to other support/lib/module files, someone writes as part of the solution?

Reply:
Nothing. People can submit them in exactly the same way that they can now.